### DISCUSSION TITLE
v2.0.0-beta.1

### DISCUSSION THREAD

**Discussion by sserrata** [PaloAltoNetworks/docusaurus-openapi-docs#488] [Announcements]
## What's Changed
* [bug] Add sass loader to theme webpack config by @sserrata in https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/pull/484
* Prepare release v2.0.0-beta.1 by @sserrata in https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/pull/487

**Full Changelog**: https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/compare/v2.0.0-beta.0...v2.0.0-beta.1

<hr /><em>This discussion was created from the release <a href='https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/releases/tag/v2.0.0-beta.1'>v2.0.0-beta.1</a>.</em>

**Comment by aj-hops** [comment]
@sserrata As I run the command `npm i docusaurus-plugin-openapi-docs` and `npm i docusaurus-theme-openapi-docs` it takes the old version, is there any way to install `2.0.0-beta` version of `docusaurus-plugin-openapi-docs` & `docusaurus-theme-openapi-docs` of this plugin by command or any other way? 

if there is any way to please let me know, thanks. 

Ref : https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/issues/323#issuecomment-1468571368

**Reply by gii-pan** [reply]
Hey @aj-hops, try to use:

```
yarn upgrade docusaurus-plugin-openapi-docs@beta
```

```
yarn upgrade docusaurus-theme-openapi-docs@beta
```

In mine it worked.

![image](https://user-images.githubusercontent.com/64155751/225717625-5dac8fe1-105f-454e-8317-f886bdd5fad5.png)

**Comment by gii-pan** [comment]
Hello @sserrata, 

Yesterday I updated to v2.0.0-beta.1 and I noticed that the type of method and the endpoint is missing.

Yours
![image](https://user-images.githubusercontent.com/64155751/225903080-6d5cec67-9f37-47ab-9e62-b30fb189fdd5.png)

Mine
![image](https://user-images.githubusercontent.com/64155751/225901809-694fce61-014e-43c1-b6f4-334bc9d972d5.png)

**Reply by sserrata** [reply]
The second screenshot indicates the that `docItemComponent` may not be properly set to use `@theme/ApiItem` for this doc instance.

**Comment by dpprowse** [comment]
I updated to v2.0.0-beta.1 today and am getting major conflicts with my install of `docusaurus-plugin-sass` which I installed as [described in the Docusaurus docs](https://docusaurus.io/docs/styling-layout#sassscss). 

![Screenshot 2023-03-23 at 11 40 28](https://user-images.githubusercontent.com/73990908/227192943-4c6c90db-f6b0-46c1-9245-0fe79a50ff62.jpg)

Commenting out the `docusaurus-plugin-sass` plugin from my `docusaurus.config.js` file resolves the problem, as mentioned in #484. However, the Docusaurus docs are instructing users to add the plugin to their config, so I'm mentioning it because this could cause issues for anyone that already uses the sass plugin for other things outside of the openapi-docs plugin.

**Reply by sserrata** [reply]
Hi @dpprowse, just a heads up that a fix is currently available in `0.0.0-beta.636`. The fix will be included in v2.0.0-beta.3 soon.
